developerportal/templates/molecules/filter-form.html

```diff
@@ -105,6 +105,21 @@ <h5>Country</h5>
     </fieldset>
   {% endif %}
 
+  {% if filters.show_search_input %}
+    <fieldset class="filter-form-section">
+      <header class="filter-form-section-header">
+        <h5>Search</h5>
+          <a href="#" class="filter-form-clear js-filter-clear" data-controls="{{SEARCH_QUERYSTRING_KEY}}">
+          <span class="icon">
+            {% include "atoms/icons/close.svg" %}
+          </span>
+          Clear
+        </a>
```

Comment:
This should be a `button type="button"` element - Golden rule: "Links take you somewhere, buttons do things" 😸

Reply:
Yep, indeed! My lame excuse is I inherited this markup / behaviour, but I guess it's on me now 😄

Reply:
Because it's a pre-existing problem, am going to split this point out into a separate ticket (but it won't get lost, I promise) so I can ship this larger piece sooner. Hope that's ok with you

Reply:
Ticketed as #1564 1564